### PR TITLE
Yawarakai component template for yawarakai-core@v1.4

### PR DESCRIPTION
**Pull request by BlueCocoa**
Signed-off-by: Cocoa <0xbbc@0xbbc.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@yawarakaijs/yawarakai-template",
```
Comment (hanamiyuna) [inline package.json]: Shouldn't contain the string of "@yawarakaijs/" here.
`"name": "yawarakai-component",` should be fine I think

File: metadata.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "displayName": "Yawarakai Component Template",
```
Comment (hanamiyuna) [inline metadata.json]: Should `displayName` and `description` both set value to `Yawarakai template component` ?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+## Installation
+
+```
+cd yawarakai
+git clone https://github.com/yawarakaijs/component-template.git Components
+```
+
+### Notice
+Please change corresponding fields in `package.json` to yours.
```
Comment (hanamiyuna) [inline README.md]: and also the metadata.json

**Review by hanamiyuna — COMMENTED** [review]
Lined out the questions

File: package.json (hunk 1 above)
Comment (BlueCocoa) [reply package.json]: Yep, will change to `"yawarakai-component"`.

File: README.md (hunk 3 above)
Comment (BlueCocoa) [reply README.md]: Yes, as well as the `metadata.json` file

**Review by hanamiyuna — APPROVED** [review]

**Pull request merged**